openedx_tagging/core/tagging/rest_api/v1/serializers.py

```diff
@@ -60,6 +100,16 @@ def to_representation(self, instance):
     def get_tags_count(self, instance):
         return instance.tag_set.count()
 
+    def get_can_tag_object(self, instance) -> Optional[bool]:
```

Comment:
Nit: Since we have the "annotations" future import, you can also just write `bool | None` which I find simpler and which doesn't require importing `Optional`